topics/go/algorithms/searches/binarysearch/binarysearch.go

```diff
@@ -45,34 +45,35 @@ func binarySearchIterative(sortedList []int, target int) (int, error) {
 // binarySearchRecursive takes the list of the sorted numbers and check it
 // with `recursive` process to find the value and return the index of array or
 // return the error if the value not found.
-func binarySearchRecursive(List []int, Target int, low int, high int) (int, error) {
+func binarySearchRecursive(sortedList []int, target int, leftIdx int, rightIdx int) (int, error) {
 
-	// 	find the middle index of array
-	mid := (low + high) / 2
+	// Calculate the middle index of the list.
+	midIdx := (leftIdx + rightIdx) / 2
 
-	// 	If low is greater than high it means the value not found,
-	//	I returned -1 and an error to show user value not found
-	for low > high {
-		return -1, fmt.Errorf("sorry value not found")
-	}
+	switch {
 
-	// 	check is List[mid] value equal with Target value than return the middle value
-	if List[mid] == Target {
-		return mid, nil
-	}
+	// check if the left index is greater than right index,
```

Comment:
This comment needs to start with a capital C.   Check if...

Reply:
I fixed it